sites/avivator/package.json

```diff
@@ -18,11 +18,9 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-dropzone": "^14.2.3",
-    "zustand": "^3.4.1"
-  },
-  "devDependencies": {
-    "@vitejs/plugin-react": "^4.2.1",
-    "vite": "^5.3.2"
+    "zustand": "^3.4.1",
+    "vite": "^5.3.2",
+    "@vitejs/plugin-react": "^4.2.1"
```

Comment:
Hmmm, I might remove TBH.  But before that why is this necessary for vercel builds? @manzt might have something more graceful up his sleeve?

Reply:
I'm alI'm also confused. Why did we get rid of devDeps for vercel? what about Vercel requires this?